<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joinpartydialog</class>
 <widget class="QDialog" name="joinpartydialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    /* --- حالت عادی دکمه --- */
    background-color: #555555; /* رنگ پس‌زمینه کمی روشن‌تر از کل صفحه */
    color: white;              /* رنگ متن */
    border: 1px solid #777777; /* یک حاشیه مشخص برای دکمه */
    padding: 8px 16px;         /* فاصله متن از لبه‌های دکمه */
    border-radius: 5px;        /* گرد کردن گوشه‌های دکمه */
    font-size: 14px;
}

QPushButton:hover {
    /* --- وقتی ماوس روی دکمه قرار می‌گیرد --- */
    background-color: #666666; /* پس‌زمینه کمی روشن‌تر می‌شود */
    border-color: #888888;
}

QPushButton:pressed {
    /* --- وقتی دکمه فشرده می‌شود --- */
    background-color: #444444; /* پس‌زمینه کمی تیره‌تر می‌شود */
    border-style: inset;       /* ایجاد حس فرورفتگی */
    padding-left: 18px;        /* حرکت جزئی متن برای حس کلیک شدن */
    padding-top: 10px;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnectToServer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ip Address</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtPort">
      <property name="placeholderText">
       <string>enter port shown of server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtIP">
      <property name="placeholderText">
       <string>enter ip shown of server</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: ..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisconnectFromServer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblServerName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>servername</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>joinpartydialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>joinpartydialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
